misk-metrics/api/misk-metrics.api

```diff
@@ -1,18 +1,14 @@
 public final class misk/metrics/FakeMetrics : misk/metrics/Metrics {
-	public fun counter (Ljava/lang/String;Ljava/lang/String;Ljava/util/List;)Lio/prometheus/client/Counter;
```

Comment:
these methods are unused?

Reply:
Not unused, but instead not overridden.

Used to be Metrics was an interface, now it's the implementation. Since there's no difference in the fake implementation, we don't need the overrides anymore.